src/Point.ts

```diff
@@ -28,23 +29,31 @@ export class Point implements IPoint {
     }
   }
 
+  protected create(): this;
+  protected create(x: number, y: number): this;
+  protected create(point: XY): this;
+  protected create(arg0: number | XY = 0, y = 0): this {
+    // @ts-expect-error constructor is not recognized
+    return new this.constructor(arg0, y);
+  }
```

Comment:
Why do we want this?

Reply:
legacy, passing new Point(x,y) is legacy. You want it dead?